### DISCUSSION TITLE
组建库请求加入Aplayer

### DISCUSSION THREAD

**Discussion by yyyytawa** [vuepress-theme-hope/vuepress-theme-hope#4567] [General]
如题....

**Comment by Mister-Hope** [comment]
I will omit feature requests like this as you are not providing any detailed information.

1. To respect open-source contributor work, you should follow the issue template and open a new issue.
2. A feature request like this should provide links about any related projects, I tried to find one: https://github.com/rRemix/APlayer, but it's not a web-based project.
3. Additionally, the feature request should list advantages or targets for a new implementation of an existing feature

**Reply by yyyytawa** [reply]
> 我将省略此类功能请求，因为您没有提供任何详细信息。
> 
> 1. 为了尊重开源贡献者的工作，您应该遵循 Issue 模板并打开一个新 Issue。
> 2. 像这样的功能请求应该提供有关任何相关项目的链接，我试图找到一个：https://github.com/rRemix/APlayer，但它不是一个基于 Web 的项目。
> 3. 此外，功能请求应列出现有功能的新实现的优势或目标

I Know about it(Tomorrow do it)
APlayer: https://aplayer.js.org/
